SwitchSpeak-iOS/SwitchSpeak-iOS/TouchGrid/TouchGrid.swift

```diff
@@ -90,6 +90,9 @@ class TouchGrid {
 	func startScanning() {
 		if !editing {
 			let scanSpeed = GlobalSettings.getUserSettings().scanSpeed.rawValue
+            scanningTimer = Timer.scheduledTimer(withTimeInterval: 0, repeats: false) { [weak self] _ in
```

Comment:
Why call `self?.selectSubTree()` in a timer with 0 delay rather than just calling it by itself?
i.e.
Why do
```swift
scanningTimer = Timer.scheduledTimer(withTimeInterval: 0, repeats: false) { [weak self] _ in
            self?.selectSubTree()
}
```
When you can just do
```swift
self?.selectSubTree()
```

Reply:
It seems like that just doing self.selectSubTree() will not work...
I also feel confused?